src/App.css

```diff
@@ -63,3 +63,7 @@ input {
 .stylebook {
   list-style: none;
 }
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/199881881-79355878-6e1e-4ee4-9f35-1069275b5714.png)

- [ ] In this component, you have created a book that's great but it would be nice if you make an array of objects for your books data (**title**, **author**) in your pages folder **Books.js** component and pass it to **Book.js** component as a prop to render it :+1: